### ISSUE TITLE
in createImage columnPixelSpacing (and rowPixelSpacing) is undefined

### ISSUE THREAD

**Issue by sculxp** [cornerstonejs/cornerstoneWADOImageLoader#190]
in createImage 
```
columnPixelSpacing: imagePlaneModule.pixelSpacing ? imagePlaneModule.pixelSpacing[1] : undefined,
rowPixelSpacing: imagePlaneModule.pixelSpacing ? imagePlaneModule.pixelSpacing[0] : undefined,
```
columnPixelSpacing and rowPixelSpacing may be undefined and in getDefaultViewport(cornerstone-core):
```
  let verticalRatio = 1;
  let horizontalRatio = 1;

  if (image.rowPixelSpacing < image.columnPixelSpacing) {
    // we believe that the row pixel is the same as css pixel
    horizontalRatio = image.columnPixelSpacing / image.rowPixelSpacing;
  } else {
    // we believe that the column pixel is the same as css pixel
    verticalRatio = image.rowPixelSpacing / image.columnPixelSpacing;
  }
  // Fit image to window
  const verticalScale = canvas.height / image.rows / verticalRatio;
  const horizontalScale = canvas.width / image.columns / horizontalRatio;
  const scale = Math.min(horizontalScale, verticalScale);
``` 
cause scale to be NAN, but metaDataProvider do return default 1/1, why not use it

**Comment by swederik** [comment]
That's a good point.

I do feel like the image loader should return undefined, since that's what it's reading from the image. But you make a point that getDefaultViewport should be smart enough to use 1 / 1 if they are undefined. It looks like this is inadvertent, since it's being checked later on: https://github.com/cornerstonejs/cornerstone/blob/master/src/internal/getDefaultViewport.js#L110

**Comment by Zaid-Safadi** [comment]
@sculxp thanks for reporting. This is now fixed in 2.2.2
